packages/lodestar/src/sync/regular.ts

```diff
@@ -68,6 +84,51 @@ export class RegularSync {
     }
   };
 
+  public receiveCommitteeAttestation = async (attestationSubnet: {attestation: Attestation; subnet: number}): 
+  Promise<void> => {
+    const attestation = attestationSubnet.attestation;
+    const subnet = attestationSubnet.subnet;
+    if (String(subnet) !== getAttestationSubnet(attestation)) {
+      return;
+    }
+    // Make sure this is unaggregated attestation
+    const aggregationBits = attestation.aggregationBits;
+    let count = 0;
+    for (let i = 0; i < aggregationBits.bitLength; i++) {
+      if (aggregationBits.getBit(i)) {
+        count++;
+      }
+    }
+    if (count !== 1) {
+      return;
+    }
+    if (await this.db.block.isBadBlock(attestation.data.beaconBlockRoot)) {
+      return;
+    }
+    const state = await this.db.state.getLatest();
+    const currentSlot = getCurrentSlot(this.config, state.genesisTime);
+    if (!(attestation.data.slot + ATTESTATION_PROPAGATION_SLOT_RANGE >= currentSlot &&
+      currentSlot >= attestation.data.slot)) {
+      return;
+    }
+    if(!isValidIndexedAttestation(this.config, state, getIndexedAttestation(this.config, state, attestation))) {
+      return;
+    }
```

Comment:
This is wrong place for validation since those validation needs to be hooked into gossipsub itself.
We shouldn't rebroadcast messages without validating them, and here is way too late to validate it.

I would suggest to just implement functions like:
- isValidAttestationGossipMessage 
- isValidAggregationAndProofGossipMessage 
etc

Might be even better to implement some class to contain those methods since you need database access for some